src/test/java/seedu/planner/commons/util/StringUtilTest.java

```diff
@@ -159,6 +161,24 @@ public void getDetails_nullGiven_throwsNullPointerException() {
         StringUtil.getDetails(null);
     }
 
+    //------------- Tests for willUnderOrOverflow -----------------------------
+
+    @Test
+    public void willUnderOrOverflow_returnsTrue() {
+        // Underflows
+        assertTrue(willUnderOrOverflow(Long.toString(Integer.MIN_VALUE - 1)));
```

Comment:
Try to test "111111111111111111111111111111111" too.

Reply:
Ok